docs/reference/builder.md

```diff
@@ -76,7 +76,7 @@ Before the Docker daemon runs the instructions in the `Dockerfile`, it performs
 a preliminary validation of the `Dockerfile` and returns an error if the syntax is incorrect:
 
     $ docker build -t test/myapp .
-    Sending build context to Docker daemon 2.048 kB
+    Sending build context to Docker daemons 2.048 kB
```

Comment:
Sorry, this is not correct, `daemon` is singular (there's only one daemon this is being sent to)